help/sources/tutorials/ui/create/advertising/linkedin-ads.md

```diff
@@ -0,0 +1,150 @@
+
+![Platform UI screenshot for Connect LinkedIn Ads account with an existing account](../../../../images/tutorials/create/linkedin-ads/existing.png?lang=en)
+
+### New account
+
+If you are creating a new account, select **[!UICONTROL New account]**, and then provide a name, an optional description, and your credentials. When finished, select **[!UICONTROL Connect to source]** and then allow some time for the new connection to establish.
+
+![Platform UI screenshot for Connect LinkedIn Ads account with a new account](../../../../images/tutorials/create/linkedin-ads/new.png?lang=en)
+
+Next, Depending on how you want the data to be grouped you must select a *pivot* value as one of the below:
+   * ACCOUNT
+   * CAMPAIGN GROUP
+   * CAMPAIGN
+   * CREATIVE
+They will enable you to restrict export to the specified Account IDs, Campaign Group IDs or Campaign IDs. Refer to [LinkedIn Reporting & ROI API Query Parameters](https://docs.microsoft.com/en-us/linkedin/marketing/integrations/ads-reporting/ads-reporting?view=li-lms-2022-07&tabs=http#query-parameters) to understand pivoting.
+
+Finally, you must provide as a mandatory input one or more of the field values below:
+| Field | Description |
+| --- | --- |
+| accountIds | Your LinkedIn account IDs. |
+| campaignGroupIds | Your LinkedIn Campaign Group IDs. |
+| campaignIds | Your LinkedIn Campaign IDs. |
+
+> [!NOTE]
+>
+> Multiple IDs can be specified by passing comma separated values.
+
+If you fail to provide a value on selecting **[!UICONTROL Select]** you will get a `Failed to parse JSON.` or `Unknown error while processing request.` error message.
+
+![Platform UI screenshot for LinkedIn Ads showing configuration](../../../../images/tutorials/create/linkedin-ads/configuration.png?lang=en)
+
+## Next steps
+
+By following this tutorial, you have established a connection to your [!DNL LinkedIn Ads] account. You can now continue on to the next tutorial and [configure a dataflow to bring data into Platform](/help/sources/ui-tutorials/dataflow/crm.md).
+
+> [!NOTE]
+>
+> On the **[!UICONTROL Scheduling]** step you need to set one of the below:
+> | Frequency | Interval |
+> | --- | --- |
+> | Day | 1 |
+> | Hour | 24 |
+
+![Platform UI screenshot for LinkedIn Ads showing scheduling](../../../../images/tutorials/create/linkedin-ads/scheduling.png?lang=en)
+
+## Additional resources
+
+### Validation {#validation}
+
+To validate that you have correctly set up the source; and [!DNL LinkedIn Ads] events are being ingested, follow the steps below:
+
+1. In the Platform UI, select View Dataflows besides the [!DNL LinkedIn Ads] card menu on the Catalog page. You can then select [!UIControl Preview dataset] to verify the data that was ingested.
+![Platform UI screenshot for LinkedIn Ads Preview Dataset](../../../../images/tutorials/create/linkedin-ads/preview-dataset.png?lang=en)
```

Comment:
Can the screenshot be updated here? There is a typo on the dataset name "Tset123"